samples/msal-angular-v2-samples/angular-b2c-sample-app/test/msa-account.spec.ts

```diff
@@ -7,7 +7,8 @@ import { BrowserCacheUtils } from "../../../e2eTestUtils/BrowserCacheTestUtils";
 
 const SCREENSHOT_BASE_FOLDER_NAME = `${__dirname}/screenshots/msa-account-tests`;
 
-describe('B2C user-flow tests (msa account)', () => {
+// Problem with MSA logins displaying a security info notice - re-enable these tests when that gets resolved
```

Comment:
You should file a tracking bug in ADO to ensure these get re-enabled.

Reply:
Will do